### ISSUE TITLE
telephony.sendSms's Future never completes

### ISSUE THREAD

**Issue by ali80** [shounakmulay/Telephony#49]
Hi, consider the following code
``` dart
 await telephony.sendSms(
        to: "PHONE_NUMBER",
        message: "Sample message",
        statusListener: (status) {
          log("status = $status");
        });
  log("sent completed"); // this line never runs
```
if I await sendSms, the last line never runs although the message is delivered successfully.

**Comment by shounakmulay** [comment]
Thanks for creating the issue @ali80 
Will fix this in next release.

**Comment by shounakmulay** [comment]
Published a new release [v0.0.9](https://pub.dev/packages/telephony/versions/0.0.9) with the fix.
